product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -26,18 +27,24 @@ You can modify your cluster by modifying:
 
 4.  You can modify the following settings on the corresponding tab of the Edit Cluster page:
 
-    | Settings                            | Tab                                                                                       |
-    | ----------------------------------- | ----------------------------------------------------------------------------------------- |
-    | Cluster name and password           | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)                 |
-    | Instance type \*                     | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab) |
-    | Networking type (public or private) \**| [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)|
-    | Database configuration parameters   | [DB Configuration](05_db_configuration_parameters)                       |
+    | Settings                                 | Tab                                                                                |
+    | ---------------------------------------- | ---------------------------------------------------------------------------------- |
+    | Cluster name and password                | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)  |
+    | Instance type \*                         | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)  |
+    | Volume properties \**                    | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab) |    | Networking type (public or private) \** | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)  |
+    | Networking type (public or private) \*** | [Cluster Settings](../../getting_started/03_create_cluster/#cluster-settings-tab)|
+    | Database configuration parameters        | [DB Configuration](05_db_configuration_parameters)                                 |
 
     \* Changing the instance type could incur higher cloud infrastructure charges.
 
-    \** If you are using Azure and previously setup a private link and want to change to a public network, you must remove the private link resources before making the change.
+    \** It can take up to a few hours to tune and resize the disks in your cluster. The volume properties are disabled and cannot be modified while this is in progress.
```

Comment:
```suggestion
    \** It can take up to six hours to tune IOPS or resize the disks of your cluster. This is due to AWS rate limiting one volume modification every six hours as explained here in [Limitations](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/modify-volume-requirements.html) section.  The volume properties are disabled and cannot be modified while this is in progress. 
```

Reply:
@dhilipkumars could we say: It can take up to six hours to tune IOPS or resize the disks of your cluster because AWS requires a cool down period after volume modifications as explained in the [Limitations](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/modify-volume-requirements.html) section. The volume properties are disabled and cannot be modified while this is in progress.

Reply:
That looks good to me.